.github/workflows/build.yml

```
@@ -1,29 +1,29 @@
-          
-      - name: install yarn dependencies        
-        run: yarn install
-      - name: prettier
-        run: yarn run format-check
-      - name: build
-        run: yarn build
\ No newline at end of file
+    BuildAndDeploy:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)